test/resources/Quotes.spec.js

```diff
@@ -191,7 +192,7 @@ describe('Quotes Resource', () => {
 
           return stripe.quotes.pdf(
             'foo_123',
-            {host: 'localhost'},
+            {host: 'localhost', maxNetworkRetries: 1},
```

Comment:
Why is it necessary to do a per-request configuration here?

Reply:
It's not really, I think it just makes the test more explicit. If we ever changed the retry behavior again the test would stay stable.

Reply:
Ah gotcha, that makes sense.